src/utils/__tests__/atRuleParamIndex.test.js

```diff
@@ -1,21 +1,18 @@
 import atRuleParamIndex from "../atRuleParamIndex"
 import postcss from "postcss"
-import test from "tape"
-
-test("atRuleParamIndex", t => {
-  t.plan(4)
 
+it("atRuleParamIndex", () => {
   atRules("@media (color) {}", atRule => {
-    t.equal(atRuleParamIndex(atRule), 7)
+    expect(atRuleParamIndex(atRule)).toBe(7)
   })
   atRules("@media  (color) {}", atRule => {
-    t.equal(atRuleParamIndex(atRule), 8)
+    expect(atRuleParamIndex(atRule)).toBe(8)
   })
   atRules("@import\n'x.css');", atRule => {
-    t.equal(atRuleParamIndex(atRule), 8)
+    expect(atRuleParamIndex(atRule)).toBe(8)
   })
   atRules("@document url-prefix(http://www.w3.org/Style/)", atRule => {
-    t.equal(atRuleParamIndex(atRule), 10)
+    expect(atRuleParamIndex(atRule)).toBe(10)
```

Comment:
We are only lucky if this works ... Each call to `atRules` is asynchronous, cannot be treated synchronously, like this.

Instead, I'd suggest something like this: 

- Modify the `atRules` util so it's less fancy. It should just return the Promise that resolves with a PostCSS result.
- Change the test to `describe` and put each assertion within its own `it` statement.
- For each assertion, return the Promise from `atRules`, following by `.then(result => /* do assertion here */)`.